File: Lecture01/MyApplication/app/src/main/java/com/codingblocks/myapplication/MainActivity.java
package com.codingblocks.myapplication;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int a = Integer.valueOf(etVar1.getText().toString());
                int b = Integer.valueOf(etVar2.getText().toString());

                int c = a + b;

                textView.setText(String.valueOf(c));
            }
        });



    }
}

File: Lecture02/MyAppKt/app/src/main/java/com/codingblocks/myappkt/MainActivity.kt
package com.codingblocks.myappkt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnAdd.setOnClickListener {
            val a = etVar1.text.toString().toInt()
            val b = etVar2.text.toString().toInt()
            val c = (a + b).toString()
            tvResult.text = c
        }
    }
}
